src/app/users/users-update/users-update.component.html

```diff
@@ -47,13 +47,14 @@
           <mat-datepicker #dp></mat-datepicker>
           <mat-error><planet-form-error-messages [control]="editForm.controls.birthDate"></planet-form-error-messages></mat-error>
         </mat-form-field>
-        <mat-form-field>
-          <mat-select i18n-placeholder placeholder="Gender" formControlName="gender" required>
-            <mat-option value="male">Male</mat-option>
-            <mat-option value="female">Female</mat-option>
-          </mat-select>
-          <mat-error><planet-form-error-messages [control]="editForm.controls.gender"></planet-form-error-messages></mat-error>
-        </mat-form-field>
+        <div>
+          <mat-radio-group i18n-placeholder formControlName="gender" required>
```

Comment:
Rather than adding `Gender` as a label please add it as a placeholder attribute of the `mat-radio-group` element